src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2308,6 +2312,163 @@ In case of PAM users, the login file should not be modified as password authenti
 NOTE: Similarly to the LDAP authentication, there is a fallback mechanism for login authentication, with the difference that it is always activated.
 A user defined in the ProActive login file always has precedence over the same user defined on the Linux system.
 
+[[_keycloak]]
+=== Keycloak-based access control
+https://www.keycloak.org/[Keycloak] is an access control solution that allows single sign-on with identity and access management aimed at modern applications and services.
+ProActive is able to connect to an existing Keycloak server, and perform user authentication and authorization against it.
+
+ProActive uses the OpenID Connect (OIDC) protocol to communicate with Keycloak and acquire an access tokens for a given user.
+It further parses the obtained access token and extracts from it the user information, including the user role.
+Based on the user role, ProActive determines the set of privileges and authorizations to be granted to the user.
+
+In order to enact ProActive access control against a Keycloak server, two steps must be accomplished: (i) deploying and configuring a Keycloak server and (ii) configuring ProActive to connect to the Keycloak server and perform access control against it.
```

Comment:
In order to perform user authentication against a Keycloak server...
(To reuse you terms from above)

Reply:
Done.